honeybee_radiance_folder/gridutil.py

```diff
@@ -0,0 +1,225 @@
+"""Utility functions for sensor grids folder."""
+import os
+import json
+
+
+def distribute_sensors(
+        input_folder, output_folder, grid_count, min_sensor_count=2000, verbose=False
+    ):
+    """Create new sensor grids folder with evenly distribute sensors.
+
+    This function creates a new folder with evenly distributed sensor grids. The folder
+    will include a ``_dist_info.json`` file which has the information to recreate the
```

Comment:
I just had a comment on this choice of name, which isn't necessarily an issue but is maybe an indication that the name could be better. `dist-info` is the same name that PyPI uses to indicate the folders that contain the distribution information for each package:

![image](https://user-images.githubusercontent.com/5567574/135855122-32ad1e82-ca22-46d8-be55-06b5b37ae375.png)

... and when I first saw the words `dist-info`, I thought that this is what you were referring to. Granted, I don't think there's anything inaccurate with using the word "distribution" to describe what you are changing here about the sensor grids but maybe "redistribution" would be a bit clearer and would also help avoid the naming conflict with PyPI distributions. After all, the sensors are already distributed across sensor grids when you input them to this function and you are just "redistributing" them into equal-sized bins. That would make the name of the file `_redist_info.json`, which is different enough that I wouldn't have gotten confused in this case.

I also trust you if you want to use another name or you think these names are in such different spaces that only a handful of people like myself will get confused by them.

Reply:
`_redist_info.json` it is!